src/lib/stitching/causality/__tests__/stitching.test.ts

```diff
@@ -97,360 +276,18 @@ describe("causality/stitching", () => {
-                  lot: {
-                    internalID: "foo",
-                  },
-                },
-              },
-            ],
-          },
-        },
-      })
-    })
-
-    it("resolves the Lot type with stitched `lot` fields", async () => {
-      const query = gql`
-        {
-          me {
-            watchedLotConnection {
-              edges {
-                node {
-                  lot {
-                    bidCount
-                    onlineAskingPrice {
-                      display
-                    }
-                  }
-                  saleArtwork {
-                    lotLabel
-                  }
-                }
-              }
-            }
-          }
-        }
-      `
-
-      // stub out the loaders necessary to assemble this data
-      const context = {
-        meLoader: jest.fn(() => Promise.resolve({ internalID: "Baz" })),
-        saleArtworksAllLoader: jest.fn(() =>
-          Promise.resolve({
-            headers: {
-              "x-total-count": 2,
-            },
-            body: [
-              {
-                _id: "foo",
-                lot_label: "lot #foo",
-              },
-              {
-                _id: "bar",
-                lot_label: "lot #bar",
-              },
-            ],
-          })
-        ),
-        causalityLoader: jest.fn(() =>
-          Promise.resolve({
-            lots: [
-              { internalID: "foo", bidCount: 2, onlineAskingPriceCents: 2200 },
-              {
-                internalID: "bar",
-                bidCount: 4,
-                onlineAskingPriceCents: 420000,
-              },
-            ],
-          })
-        ),
-        saleArtworkRootLoader: jest.fn(() =>
-          Promise.resolve({ currency: "USD" })
-        ),
-      }
-
-      const data = await runAuthenticatedQuery(query, context)
-
-      expect(data).toEqual({
-        me: {
-          watchedLotConnection: {
-            edges: [
-              {
-                node: {
-                  saleArtwork: { lotLabel: "lot #foo" },
-                  lot: {
-                    bidCount: 2,
-                    onlineAskingPrice: {
-                      display: "$22",
-                    },
-                  },
-                },
-              },
-              {
-                node: {
-                  saleArtwork: { lotLabel: "lot #bar" },
-                  lot: {
-                    bidCount: 4,
-                    onlineAskingPrice: {
-                      display: "$4,200",
-                    },
-                  },
-                },
-              },
-            ],
-          },
-        },
-      })
+  describe("Me", () => {
+    it("extends Me type with #auctionsLotStandingConnection", async () => {
+      const { getFields } = await useCausalityStitching()
+      expect(await getFields("Me")).toContain("auctionsLotStandingConnection")
     })
   })
 
-  describe("auctionsLotStandingConnection", () => {
```

Comment:
All of this test code can be removed, because we no longer need to assert against what causality's stitched pass-through returns as we're now just making a simple query and attaching it to the `Lot` type.